.ci-self/esy-build-steps.yml

```diff
@@ -5,21 +5,21 @@ steps:
   - template: utils/use-esy.yml
   - script: "npm install -g pesy@next"
     displayName: "install pesy" 
-  - script: "pesy --fetch-cache=false --directory $(Pipeline.Workspace)/hello-reason --template=reason-native-web/morph-graphql-server-pesy-template#$(Build.SourceBranch)"
+  - script: "pesy --fetch-cache=false --directory $(Agent.HomeDirectory)/hello-reason --template=reason-native-web/morph-graphql-server-pesy-template#$(Build.SourceBranch)"
     displayName: "Initialize pesy template"
   - script: "esy test"
     displayName: "esy test"
-    workingDirectory: $(Pipeline.Workspace)/hello-reason
+    workingDirectory: $(Agent.HomeDirectory)/hello-reason
   - template: utils/publish-build-cache.yml
-  - task: PublishBuildArtifacts@1  
-    displayName: 'Upload project directory'
-    # TODO: The CI Build caches are pulled down by the last successful buildID
-    # for the target branch.
-    inputs:
-        pathToPublish: '$(Pipeline.Workspace)'
-        artifactName: 'project-snapshot-$(Agent.OS)'
-        parallel: true
-        parallelCount: 8
+  # - task: PublishBuildArtifacts@1  
```

Comment:
I used this task to download and investigate what went wrong. Leaving it here in case you need it

Reply:
Great